### PR TITLE
tests: load test skeleton

### PR DESCRIPTION
**Pull request by alanchchen**

### CODE REVIEW

File: benchmark/kubernetes/validator/values.yaml [hunk 1]
```diff
@@ -27,26 +30,26 @@ ethereum:
     enabled: true
     path: ""
   rpc:
-    enabled: true
+    enabled: false
     addr: 0.0.0.0
     port: 8545
-    api: "eth,net,web3,personal"
+    api: "eth,net,web3,personal,mining,admin"
```
Comment (alanchchen) [inline benchmark/kubernetes/validator/values.yaml]: miner

File: benchmark/kubernetes/validator/values.yaml [hunk 2]
```diff
@@ -27,26 +30,26 @@ ethereum:
...
     corsdomain: "*"
   ws:
-    enabled: false
+    enabled: true
     addr: 0.0.0.0
     port: 8546
-    api: "eth,net,web3,personal"
+    api: "eth,net,web3,personal,mining,admin"
```
Comment (bailantaotao) [inline benchmark/kubernetes/validator/values.yaml]: miner

File: common/utils.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func GenerateRandomDir() (string, error) {
 
 func GenerateKeys(num int) (keys []*ecdsa.PrivateKey, nodekeys []string, addrs []common.Address) {
 	for i := 0; i < num; i++ {
-		nodekey := RandomHex()
+		nodekey := RandomHex()[2:]
```
Comment (bailantaotao) [inline common/utils.go]: `[2:]` Is it to skip '0x' char?

File: cmd/utils/utils.go [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+		keys = append(keys, key)
+
+		addr := crypto.PubkeyToAddress(key.PublicKey)
+		addrs = append(addrs, addr)
+	}
+
+	return keys, nodekeys, addrs
+}
+
+// func GenerateKeys(num int) (keys []*ecdsa.PrivateKey, addrs []common.Address) {
```
Comment (tailingchen) [inline cmd/utils/utils.go]: remove it?

File: tests/table.go [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+func PCaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, true, false)
+	return true
+}
+
+/*
+You can mark a table as pending with `XCaseTable`.
+*/
+func XCaseTable(description string, itBody interface{}, entries ...tableEntry) bool {
+	caseTable(description, itBody, entries, true, false)
```
Comment (bailantaotao) [inline tests/table.go]: Should the last parameter is true?
caseTable(description, itBody, entries, true, `false`)

File: tests/table_entry.go [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+*/
+func PCase(description string, parameters ...interface{}) tableEntry {
+	return tableEntry{description, parameters, true, false}
+}
+
+/*
+You can mark a particular entry as pending with XEntry.  This is equivalent to XIt.
+*/
+func XCase(description string, parameters ...interface{}) tableEntry {
+	return tableEntry{description, parameters, true, false}
```
[the remark this answers is not in the document]
Comment (alanchchen) [reply tests/table_entry.go]: The last `bool` means focused. AFAIK, it's set to `true` only by FCase.

File: tests/table.go (hunk 5 above)
Comment (alanchchen) [reply tests/table.go]: The last bool means focused. AFAIK, it's set to true only by FCase.

File: common/utils.go [hunk 7]
```diff
@@ -35,6 +39,14 @@ const (
 	nodekeyFileName  = "nodekey"
 )
 
+func GenerateIPs(num int) (ips []string) {
```
Comment (tailingchen) [inline common/utils.go]: How about to defined subnet `10.1.1.xxx` as const value and pass it as a param of `GenerateIPs `?
```
func GenerateIPs(subnet string, num int) (ips []string){
    ....
}
```

**Pull request merged**